src/Api/Data/StoryblokLink.php

```diff
@@ -15,5 +18,15 @@ public function __construct (array $data)
 		$this->id = $data["id"];
 		$this->uuid = $data["uuid"];
 		$this->slug = $data["slug"];
+		$this->isFolder = $data["is_folder"];
```

Comment:
Proposal: Rename `isFolder` to `hasFolderTarget`, or is that too confusing or even too verbose?